CONTRIBUTING.rst

```diff
@@ -161,6 +156,38 @@ Clone the repository
    ``strawberry-sqlalchemy-mapper`` comes with a lot of hooks configured to automatically help the
    developer to check the code being written.
 
+
+Method 2: Dev Container setup
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Install `Dev Containers`_.
+
+#. Press ``Ctrl+Shift+P`` to open the command tab and enter ``Dev Containers: Open Folder in Container...``.
+
+#. Select the root directory of this repository.
+   Dev Containers will now install the environment dependencies as well as |pre-commit|_
+
+#. When installation is complete, press ``Ctrl+Shift+P`` to open the command tab and enter ``Python Select Interpreter``
+
+#. Select the environment labeled 'poetry'.
+
+
+Code Contributions
+==================
+
+.. todo:: Please include a reference or explanation about the internals of the project.
+
+   An architecture description, design principles or at least a summary of the
+   main concepts will make it easy for potential contributors to get started
+   quickly.
+
+Submit an issue
+---------------
+
+Before you work on any non-trivial code contribution it's best to first create
+a report in the `issue tracker`_ to start a discussion on the subject.
```

Comment:
The var `issue tracker` returns a incorrect URL: `https://github.com/<USERNAME>/strawberry-sqlalchemy-mapper/issues`, please lets update this to the correct one: https://github.com/strawberry-graphql/strawberry-sqlalchemy/issues